config/src/config/indexer_grpc_config.rs

```diff
@@ -36,6 +38,12 @@ pub struct IndexerGrpcConfig {
 
     /// Number of transactions returned in a single stream response
     pub output_batch_size: u16,
+
+    /// Number of table info parsing tasks to fan out
+    pub parser_task_count: u16,
+
+    /// Number of transactions each table info parser will process
+    pub parser_batch_size: u16,
```

Comment:
Why are we reusing grpc config for this? This should be its own config since it's a separate runtime controlled by a separate flag.